### PR TITLE
Feature: doctor notifications

### PR DESCRIPTION
**Pull request by mr3nz1**
### What does this pull request do?
This pull request allows the doctor to receive notifications of important actions both from the patient or through their actions.

### Description
- Real-time notifications

### CODE REVIEW

**Review by munezeromicha — APPROVED** [review]
Good Job!

**Review by Irirwanirira — APPROVED** [review]
Good job

File: src/app/dashboard/notifications/page.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        {notifications.map((notification) => {
+          return (
+            <NotificationListing
+              key={notification.id}
+              //   title={notification.title}
+              //   description={notification.description}
+              //   createdAt={notification.createdAt}
+              //   doctorId={notification.doctorId}
+              //   id={id}
+              //   patientId={notification.}
```
Comment (Nkbtemmy) [inline src/app/dashboard/notifications/page.tsx]: Remove this codes if you don't need them !!

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]

File: src/app/dashboard/notifications/page.tsx (hunk 1 above)
Comment (Irirwanirira) [reply src/app/dashboard/notifications/page.tsx]: done

**Review by Nkbtemmy — APPROVED** [review]

**Pull request merged**